src/context/environment.ts

```diff
@@ -193,7 +193,7 @@ export class EnvironmentContext {
    * @returns string indicating the feature this environment relates to, if not feature environment returns an empty string
    */
   static getFeatureInfo(scope: Construct): string {
-    if (EnvironmentContext.isFeature(scope) !== false) return "";
+    if (EnvironmentContext.isFeature(scope) !== true) return "";
```

Comment:
suggest: `isFeature` always returns `boolean`, so following the Sonar rule [_Boolean literals should not be used in comparisons_](https://next.sonarqube.com/sonarqube/coding_rules?open=typescript%3AS1125&rule_key=typescript%3AS1125) might be more readable.
```suggestion
    if (!EnvironmentContext.isFeature(scope)) return "";
```